flow/layers/performance_overlay_layer_unittests.cc

```diff
@@ -22,20 +24,21 @@ constexpr int kMockedTimes[] = {17, 1,  4,  24, 4,  25, 30, 4,  13, 34,
                                 14, 0,  18, 9,  32, 36, 26, 23, 5,  8,
                                 32, 18, 29, 16, 29, 18, 0,  36, 33, 10};
 
-// Relative to the flutter/src/engine/flutter directory
-const char* kGoldenFileName = "performance_overlay_gold.png";
-
-// Relative to the flutter/src/engine/flutter directory
-const char* kNewGoldenFileName = "performance_overlay_gold_new.png";
-
-TEST(PerformanceOverlayLayer, Gold) {
-  const std::string& golden_dir = flutter::GetGoldenDir();
+static std::string GetGoldenFilePath(int refresh_rate, bool is_new) {
+  std::stringstream ss;
   // This unit test should only be run on Linux (not even on Mac since it's a
   // golden test). Hence we don't have to worry about the "/" vs. "\".
-  std::string golden_file_path = golden_dir + "/" + kGoldenFileName;
-  std::string new_golden_file_path = golden_dir + "/" + kNewGoldenFileName;
+  ss << flutter::GetGoldenDir() << "/"
```

Comment:
At some point, we should start using the fixtures mechanism. Later is fine. That way, the test can discover its assets without needing the specification of extra paths at the command line.